<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CaptureSrcUI</class>
 <widget class="QDialog" name="CaptureSrcUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>226</width>
    <height>65</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>226</width>
    <height>65</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>226</width>
    <height>65</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="1" column="0">
    <widget class="QFrame" name="CaptureSrcUIBakFrame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>9</number>
      </property>
      <property name="topMargin">
       <number>9</number>
      </property>
      <property name="bottomMargin">
       <number>9</number>
      </property>
      <item>
       <widget class="ButtonWdg" name="DesktopBtnWdg" native="true">
        <property name="minimumSize">
         <size>
          <width>69</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>69</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="ButtonWdg" name="SoftWareBtnWdg" native="true">
        <property name="minimumSize">
         <size>
          <width>69</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>69</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="ButtonWdg" name="RegionBtnWdg" native="true">
        <property name="minimumSize">
         <size>
          <width>69</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>69</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ButtonWdg</class>
   <extends>QWidget</extends>
   <header>buttonwdg.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../VhallLive/project/VhallLive.qrc"/>
 </resources>
 <connections/>
</ui>
